src/main/java/duke/Task/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package duke.Task;
+
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {
+    private final LocalDateTime dateTime;
+
+    /**
+     * Constructs a Deadline object to represent a Deadline Task
+     * @param task description of task
+     * @param dateTime LocalDateTime object that represents the deadline
+     */
+    public Deadline(String task, LocalDateTime dateTime) {
+        super(task);
+        this.dateTime = dateTime;
+    }
+
+    /**
+     * Returns a String in the format of "dd MM yyyy hh mm"
+     * @return String that represents the date and time of the Deadline Task
+     */
+    public String getDateTime() {
```

Comment:
Is there a more elegant way to do this? You may want to check on this link for more information :) https://stackoverflow.com/questions/3469507/how-can-i-change-the-date-format-in-java